### PR TITLE
Misc node log fixes

### PR DESCRIPTION
**Pull request by jianoaix**
## Why are these changes needed?
Make sure each exit of batch processing has a log

## Checks

- [x] I've made sure the lint is passing in this PR.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: node/node.go [hunk 1]
```diff
@@ -350,6 +350,7 @@ func (n *Node) ProcessBatch(ctx context.Context, header *core.BatchHeader, blobs
 		// revert all the keys for that batch.
 		result := <-storeChan
 		if result.keys != nil {
+			log.Debug("Batch validation failed, rolling back the key/value entries stored in database", "number of entires", len(*result.keys), "batchHeaderHash", batchHeaderHash)
```
Comment (dmanc) [inline node/node.go:353]: Can we use batchHeaderHashHex instead

Comment (dmanc) [reply node/node.go:353]: Same comment for below logs

File: node/node.go [hunk 2]
```diff
@@ -363,23 +364,23 @@ func (n *Node) ProcessBatch(ctx context.Context, header *core.BatchHeader, blobs
...
+		log.Error("Store batch failed", "batchHeaderHash", batchHeaderHash, "err", result.err)
 		return nil, err
 	}
 	if result.keys != nil {
 		n.Metrics.AcceptBatches("stored", batchSize)
 		n.Metrics.ObserveLatency("StoreChunks", "stored", result.latency)
-		n.Logger.Debug("Store batch took", "duration:", time.Duration(result.latency*float64(time.Millisecond)))
+		n.Logger.Debug("Store batch succeeded", "batchHeaderHash", batchHeaderHash, "duration:", time.Duration(result.latency*float64(time.Millisecond)))
+	} else {
+		n.Logger.Warn("Store batch skipped because the batch already exists in the store", "batchHeadherHash", batchHeaderHash)
```
Comment (dmanc) [inline node/node.go:375]: typo: batchHeadherHash -> batchHeaderHash

**Review by dmanc — APPROVED** [review]
lgtm just some minor stuff

File: node/node.go (hunk 2 above)
Comment (jianoaix) [reply node/node.go]: Done.

**Pull request merged**